cpuinfo_test.go

```diff
@@ -402,3 +466,16 @@ func TestCPUInfoParseRISCV64(t *testing.T) {
 		t.Errorf("want ModelName %v, have %v", want, have)
 	}
 }
+
+func TestCPUInfoParseLOONG64(t *testing.T) {
+	cpuinfo, err := parseCPUInfoLoong([]byte(cpuinfoLoong64))
+	if err != nil || cpuinfo == nil {
+		t.Fatalf("unable to parse loongarch cpu info: %v", err)
+	}
+	if want, have := 4, len(cpuinfo); want != have {
+		t.Errorf("want number of processors %v, have %v", want, have)
+	}
+	if want, have := "Loongson-64bit", cpuinfo[1].CPUFamily; want != have {
+		t.Errorf("want ModelName '%v', have '%v'", want, have)
+	}
```

Comment:
`want CPUFamily`. Also please check all fields you parsed, which means adding the `ModelName` assertion back.

Reply:
Thank you for reviewing, I have modified the code